### ISSUE TITLE
TypeError: `Cannot convert undefined or null to object` in `waitForAllAttributesAndContinue`

### ISSUE THREAD

**Issue by dengribar** [meteor/blaze#456]
### Summary
👋 After updating Blaze from **2.7.1** to **2.8.0** in our Meteor (v**2.14**) app, we got `Cannot convert undefined or null to object` error in `waitForAllAttributesAndContinue` function ([ref](https://github.com/meteor/blaze/blob/adc11c9a8bd5d638100d8ff7fb6dc3dc095671e4/packages/blaze/materializer.js#L100)):

<img width="558" alt="image 28" src="https://github.com/meteor/blaze/assets/111731892/7c8e9b51-8e4b-473c-b600-b08aa8d76dd8">

In our case it happens when an HTML element has only one attribute and it's value is calculated by a Template helper function that returns `undefined`.

### Steps to reproduce
1. Create a Blaze app with command: `meteor create --blaze simple-todos-blaze --prototype`
2. Add `disabled` helper to `hello` Template in `main.js` module:
```js
Template.hello.helpers({
  disabled() {
    return undefined;
  },
```
3. Add `disabled` attribute to `button` element in `hello` Template in `main.html` module:
```js
<template name="hello">
  <button {{disabled}}>Click Me</button>
```
4. Run the app

#### Actual behaviour
The app crashes in browser with the described error

#### Expected behaviour
The element should be rendered with no attributes

**Comment by jankapunkt** [comment]
Should be easily fixed by making `attrs = {}`

**Comment by jankapunkt** [comment]
The issue actually comes from `Blaze._expandAttributes` which should return and empty Object, instead of null, when no atts were found to be expandable. Otherwise multiple functions will fail (including `waitForAllAttributesAndContinue`)

PR is incoming

**Comment by dengribar** [comment]
This issue is fixed in Blaze 2.9.0, thanks to everyone involved! 🥇
